tfe/tfe_sweeper_test.go

```diff
@@ -0,0 +1,59 @@
+package tfe
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/go-tfe"
+	"log"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestMain(m *testing.M) {
+	resource.TestMain(m)
+}
+
+func getOrgSweeper(name string) *resource.Sweeper {
+	return &resource.Sweeper{
+		Name: name,
+		F: func(ununsed string) error {
+			if os.Getenv("TFE_HOSTNAME") == "" && (os.Getenv("TFE_TOKEN") == "") {
+				return fmt.Errorf("must provide environment variables TFE_HOSTNAME and TFE_TOKEN")
+			}
+			hostname := os.Getenv("TFE_HOSTNAME")
+			token := os.Getenv("TFE_TOKEN")
+			insecure := os.Getenv("TFE_INSECURE") != ""
+
+			client, err := getClient(hostname, token, insecure)
+			if err != nil {
+				return err
+			}
+
+			ctx := context.TODO()
+			orgList, err := client.Organizations.List(ctx, tfe.OrganizationListOptions{})
+			if err != nil {
+				return err
+			}
+			for _, org := range orgList.Items {
+				log.Printf("[DEBUG] Testing if org %s starts with tst-terraform or named terraform-updated", org.Name)
+				if strings.HasPrefix(org.Name, "tst-terraform") || org.Name == "terraform-updated" {
+					log.Printf("[DEBUG] deleting org %s", org.Name)
+					err = client.Organizations.Delete(ctx, org.Name)
+					if err != nil {
+						return err
+					}
```

Comment:
should the error be returned here? the way i'm reading this makes me thing we would return the first time we encountered an error deleting an organization in our list. it seems like we would want to continue through the list instead. the code suggestion feature is not working for me today so hopefully this makes sense 😅 
```
					err = client.Organizations.Delete(ctx, org.Name)
					if err != nil {
						log.Printf("Error destroying organization %s during sweep: %s", org.Name, err)
					}
```